README.md

```diff
@@ -9,9 +9,9 @@ This is a simple CI server for the [Assignment #2: Continuous Integration](https
 
 ## Running the project
 - Clone the repo and make sure you are inside the CI subfolder
-- Run **mvn test** to confirm that everything is working
+- Run `mvn test` to confirm that everything is working, the "githubPAT" file is required.
```

Comment:
I think the user has to also know where to put the githubPAT file I mention that is supposed to be in the root directory of the repo